pkg/cloudproviders/awsprovider/templates/controlPlane.yaml

```diff
@@ -0,0 +1,470 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+Description: CloudFormation Template to create the Control Plane Configuration
+
+Parameters:
+  CIDR:
+    Description: The Primary CIDR of the VPC being created.
+    Type: String
+  ImageID:
+    Description: Image ID
+    Type: String
+  InstanceType:
+    Description: Instance Type
+    Type: String
+  InstanceCount:
+    Description: Instance Count
+    Type: Number
+    Default: 3
+  IgnitionScriptS3URL:
+    Description: The Object URL for the ignition script.
+    Type: String
+
+Resources:
+  CPVPC:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: !Ref 'CIDR'
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      Tags:
+        - Key: "Name"
+          Value: !Join ["-", [!Ref 'AWS::StackName', "cp"]]
```

Comment:
- generally, what'll be the length of this stackname?
- if this reduces our visibility to view the network in AWS Console VPC Resource Maps, let's have this as a suffix (same for the rest)

Reply:
- maximum length of the stack name can be 128 characters
- we can have the stack name as the clusterID which will be unique 
- yes I will add it as a suffix